src/sentry/web/forms/accounts.py

```diff
@@ -163,6 +163,16 @@ class RegistrationForm(forms.ModelForm):
         widget=forms.TextInput(attrs={'placeholder': 'you@example.com'}))
     password = forms.CharField(
         widget=forms.PasswordInput(attrs={'placeholder': 'something super secret'}))
+    subscribe = forms.BooleanField(
+        label=_('Subscribe to product updates newsletter'),
```

Comment:
How about: 'Subscribe to Sentry's monthly release updates'?

I'd rather have more people subscribed and be limited to a monthly update than less people subscribe and us send tons of emails, especially to the OSS community.